### PR TITLE
Fixes #306 Display a list of registered repositories in User's dashboard

### LINKED ISSUE #306: Display a list of registered repositories in User's dashboard
### This is a
- [ ] Bug report
- [x] Feature request

### Description

A user may not keep track of the repositories he or she have registered. Showing a list of registered repositories will help the user know of their registered repositories.

### PR DESCRIPTION
**Pull request by imujjwal96**
## Description

* Updated Repositories Model
* Made changes conforming to changes made 
* Separately retrieve user owned repositories and organization owned repostories
* Added dummy buttons for Enabling/Disabling and Delete beside each repository
* Encrypted access token
## Related Issue

issue: #306 
## Motivation and Context

A user may not keep track of the repositories he or she have registered. Showing a list of registered repositories will help the user know of their registered repositories.

## How Has This Been Tested?

Test at: https://yaydoc6.herokuapp.com/
## Screenshots (if appropriate):
image

## Types of changes

- [ ] Bug fix 
- [x] New feature

## Checklist:

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only one commit per issue.

### CODE REVIEW

File: views/dashboard.jade [hunk 1]
```diff
@@ -40,21 +40,64 @@ block content
...
                 input.form-control#search_bar(placeholder="Enter project name")
                 span.input-group-btn
                   button.btn.btn-default#search(type="button") Search
             .form-group#search_result
             button.btn.btn-default#btnRegister(disabled) Register
+      hr
+      h3 Registered Repositories
+      ul(style="list-style:none")
+        li
+          h4(style="font-weight: 700") Ujjwal Bhardwaj
```
Comment (pri22296) [inline views/dashboard.jade]: I guess you forgot to remove your name after testing?

Comment (imujjwal96) [inline views/dashboard.jade]: >.<

Comment (imujjwal96) [inline views/dashboard.jade]: @pri22296 updated

Comment (pri22296) [inline views/dashboard.jade]: Also update the deployment

Comment (imujjwal96) [inline views/dashboard.jade]: @pri22296 updated!

Comment (pri22296) [inline views/dashboard.jade]: Here also application error

Comment (imujjwal96) [inline views/dashboard.jade]: Updated!

**Comment by pri22296** [comment]
screenshot 191

It shows no repository registered when repo is already registered.

**Comment by imujjwal96** [comment]
@pri22296 That is because a webhook is registered for this repository. You will have to manually remove the webhook from repository's settings. I'll fix this in another PR

**Pull request merged**